test/cpp/util/test_credentials_provider.h

```diff
@@ -29,8 +29,10 @@ namespace grpc {
 namespace testing {
 
 const char kInsecureCredentialsType[] = "INSECURE_CREDENTIALS";
-// For real credentials, like tls/ssl, this name should match the AuthContext
-// property "transport_security_type".
+const char kTls12CredentialsType[] = "tls12";
```

Comment:
Please add comments that help explain the difference between the types that you added vs. the original `kTlsCredentialsType`.